includes/admin/class-wc-admin-setup-wizard.php

```diff
@@ -132,6 +132,21 @@ protected function should_show_mailchimp() {
 		return current_user_can( 'install_plugins' );
 	}
 
+	/**
+	 * Should we show the Facebook install option?
+	 * True only if the user can install plugins,
+	 * and up until April 30, 2019.
+	 *
+	 * @return boolean
+	 */
+	protected function should_show_facebook() {
+		$end_date_facebook_recommendation = new DateTime( '16 April 2019' );
```

Comment:
Little bit picky but this would be faster if a timestamp was hardcoded (int) rather than using a datetime object?

Reply:
Yeh, I added that for clarity. Happy to change it with a comment for the date instead.

Reply:
(I'll update when we know the actual end date)

Reply:
I updated the date, but opted to keep `DateTime`. The reason is because it's more readable and does not cause a performance issue since `should_show_facebook` is only called twice - both times on one page of the OBW.